app/[locale]/job/FormPayment.tsx

```diff
@@ -1,13 +1,15 @@
 'use client'
 
-import ErrorSummary from "@/app/components/ErrorSummary"
```

Comment:
this is reordering of `import`s to be more standard (react->get data->render pages)